File: backend/src/main/java/com/ameri/objects/beans/adminBeans/ProvisionalBean.java
package com.ameri.objects.beans.adminBeans;

import java.util.List;

public class ProvisionalBean {

    List<AdminBeans> adminBeansList;
    List<GananciaBeans> gananciaBeans;

    public ProvisionalBean(){}

    public ProvisionalBean(List<AdminBeans> adminBeansList, List<GananciaBeans> gananciaBeans) {
        this.adminBeansList = adminBeansList;
        this.gananciaBeans = gananciaBeans;
    }

    public List<AdminBeans> getAdminBeansList() {
        return adminBeansList;
    }

    public void setAdminBeansList(List<AdminBeans> adminBeansList) {
        this.adminBeansList = adminBeansList;
    }

    public List<GananciaBeans> getGananciaBeans() {
        return gananciaBeans;
    }

    public void setGananciaBeans(List<GananciaBeans> gananciaBeans) {
        this.gananciaBeans = gananciaBeans;
    }
}

File: frontend/src/objects/classes/beans/adminBeans/ProvisionalBean.ts
import {AdminBeans} from "./AdminBeans";
import {GananciaBeans} from "./GananciaBeans";

export class ProvisionalBean{

  adminBeansList: Array<AdminBeans>;
  gananciaBeans: Array<GananciaBeans>;


  constructor(adminBeansList: Array<AdminBeans>, gananciaBeans: Array<GananciaBeans>) {
    this.adminBeansList = adminBeansList;
    this.gananciaBeans = gananciaBeans;
  }
}
